pom.xml

```
@@ -866,6 +781,201 @@
   </distributionManagement>
 
   <profiles>
+    <profile>
+      <!--
+       The 'no-jenkins-bom' profile captures the properties & dependencies when _not_ using the Jenkins BOM:
+        https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml
+       When the BOM is not being used, this profile is adding back the <dependencyManagement /> entries that were
+       moved to the BOM.
+       The profile is enabled by default; if you _are_ using the Jenkins BOM then disable this profile by defining the
+       'use-jenkins-bom' system property.
+      -->
+      <id>no-jenkins-bom</id>
+      <activation>
+        <property>
+          <name>!use-jenkins-bom</name>
+        </property>
+      </activation>
+      <properties>
+        <slf4jVersion>1.7.26</slf4jVersion>
```

this should be 1.7.25.  ref: https://github.com/jenkinsci/jenkins/blob/jenkins-2.138.4/pom.xml#L93

```suggestion
        <slf4jVersion>1.7.25</slf4jVersion>
```

---

@oleg-nenashev I think this was a regression that dependabot introduced.  the version needs to match jenkins core that you are targeting, otherwise you can use a newer API that what you get at runtime and then *boom*.

I will not merge this, but if you want to merge it and then release that is fine with me,